addons/website_event_track_session/models/event_track.py

```diff
@@ -139,7 +139,10 @@ def _get_track_suggestions(self, restrict_domain=None, limit=None):
             lambda track:
                 (track.is_published,
                  track.is_track_live,
```

Comment:
Not sure, do we need this "track.is_track_live" ? as we now use remaining time ...